tests/VolleyManagement.UnitTests/Mvc/ViewModels/TeamMvcViewModelBuilder.cs

```diff
@@ -14,27 +14,27 @@ internal class TeamMvcViewModelBuilder
         /// <summary>
         /// Holds test tournament view model instance
         /// </summary>
-        private TeamViewModel _teamViewModel;
+        private readonly TeamViewModel _teamViewModel;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="TeamMvcViewModelBuilder"/> class
         /// </summary>
         public TeamMvcViewModelBuilder()
         {
-            _teamViewModel = new TeamViewModel()
-            {
+            _teamViewModel = new TeamViewModel() {
                 Id = 1,
                 Name = "Name",
                 Coach = "Coach",
                 Achievements = "Achievements",
-                Captain = new PlayerNameViewModel() { Id = 1, FirstName = "First", LastName = "Player" },
+                Captain = new PlayerNameViewModel() { Id = 100, FirstName = "First", LastName = "Player" },
```

Comment:
Why 100?

Reply:
because captain has to have special id in some of test cases, and in some cases we set another player as captain with id 1 and it brokes